docs/guides/src/main/operator/installation.adoc

```diff
@@ -5,14 +5,48 @@
 
 <@tmpl.guide
 title="Keycloak Operator Installation"
-summary="How to install the Keycloak Operator on Kubernetes">
-
-TODO: https://github.com/keycloak/keycloak/issues/10786
+summary="How to install the Keycloak Operator on Kubernetes and OpenShift">
 
 == Keycloak Operator Installation
+In this guide we will show how to install the Keycloak Operator in your Kubernetes or OpenShift cluster.
+
+=== OLM Installation
+
+You can install the Keycloak Operator in Kubernetes environment is to use Operator Lifecycle Manager (OLM).
+Before moving on, make sure you have OLM installed, if not follow this https://github.com/operator-framework/operator-lifecycle-manager/blob/master/doc/install/install.md#install-a-release[guide].
+
+To install the Keycloak Operator follow the https://olm.operatorframework.io/docs/tasks/install-operator-with-olm/[instructions].
+In the default Catalogs the Keycloak Operator is named `keycloak-operator` and make sure to use the `candidate` channel.
+
+==== OpenShift UI
+
+On OpenShift you can use a conveninet UI on top of OLM.
+Navigate `Home` → `Operators` → `OperatorHub` in the menu on the left side of the OpenShift Console.
+Then, focus on the search input box and type "keycloak":
+
+image::{generatedGuideImages}/select-operator.jpeg["Select the Keycloak Operator in the UI"]
 
-=== OpenShift Installation
+Next, navigate to Keycloak Operator and click on it. Next, follow the instructions on the screen, make sure you are installing from the `candidate` channel:
```

Comment:
```suggestion
Select the Keycloak Operator from the list of results. After that, follow the instructions on the screen. Make sure you are installing from the `candidate` channel:
```

Reply:
not sure if the "candidate" sentence and pic should come before this, because when reading it it may be that i would have followed instructions on the screen to come back to this guide and only later on see that i installed from wrong channel. makes sense for you?